### PR TITLE
Small improvements

### PR DESCRIPTION
**Pull request by alien11689**

### CODE REVIEW

**Comment by tdziurko** [comment]
1. I am not a bit fan of finals in the code, but... new project, new approach, a small step outside of my comfort zone of coding habits :)
2. Travis - awesome

File: src/test/groovy/pl/tomaszdziurko/jvm_bloggers/mailing/BlogSummaryMailGeneratorSpec.groovy [hunk 1]
```diff
@@ -17,11 +17,8 @@ class BlogSummaryMailGeneratorSpec extends Specification {
     def "Should populate template with posts data"() {
         given:
             List<BlogPost> newPosts = [stubBlogPost("New blog post", "http://example.com", "Piotr Nowak")]
-        when:
-            String mail = blogSummaryMailGenerator.generateSummaryMail(newPosts, 7)
-        then:
-            mail != null
-            noExceptionThrown()
+        expect:
+            blogSummaryMailGenerator.generateSummaryMail(newPosts, 7) != null
```
Comment (marcingrzejszczak) [inline src/test/groovy/pl/tomaszdziurko/jvm_bloggers/mailing/BlogSummaryMailGeneratorSpec.groovy]: You can even write: `blogSummaryMailGenerator.generateSummaryMail(newPosts, 7)`

Comment (alien11689) [inline src/test/groovy/pl/tomaszdziurko/jvm_bloggers/mailing/BlogSummaryMailGeneratorSpec.groovy]: Yeah... I know, but I prefer when assert is more meaningful.

2015-09-19 17:47 GMT+02:00 Marcin Grzejszczak <notifications@github.com>:

> In
> src/test/groovy/pl/tomaszdziurko/jvm_bloggers/mailing/BlogSummaryMailGeneratorSpec.groovy
> <https://github.com/tdziurko/jvm-bloggers/pull/28#discussion_r39920768>:
>
> > @@ -17,11 +17,8 @@ class BlogSummaryMailGeneratorSpec extends Specification {
> >      def "Should populate template with posts data"() {
> >          given:
> >              List<BlogPost> newPosts = [stubBlogPost("New blog post", "http://example.com", "Piotr Nowak")]
> > -        when:
> > -            String mail = blogSummaryMailGenerator.generateSummaryMail(newPosts, 7)
> > -        then:
> > -            mail != null
> > -            noExceptionThrown()
> > +        expect:
> > +            blogSummaryMailGenerator.generateSummaryMail(newPosts, 7) != null
>
> You can even write: blogSummaryMailGenerator.generateSummaryMail(newPosts,
> 7)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tdziurko/jvm-bloggers/pull/28/files#r39920768>.
>



-- 
Pozdrawiam,
Dominik Przybysz

Comment (marcingrzejszczak) [inline src/test/groovy/pl/tomaszdziurko/jvm_bloggers/mailing/BlogSummaryMailGeneratorSpec.groovy]: +1

**Pull request merged**